### PR TITLE
Revamped previous User endpoints, added File endpoints, modified Read.me, installed new node modules

### PR DESCRIPTION
**Pull request by henryfan03**
**### General Description**
- File upload to AWS S3 has been implemented.
- User endpoints have been revamped to include better logging messages.
- Router now supports the usage of more than one type of endpoint (users and files). Look at routes/index.js for reference
- Readme has been updated to include information about the structure of S3 object notation and our file naming convention.
- dotenv configuration was not included in server.js, this change includes that fix.
- New directory utilities for organizing files that are not routes nor data models. (you will find uploadImage.js)
- uploadImage.js: Includes S3 client configuration, multer configuration, implementation of our S3 naming convention, error handling, and middleware.

### CODE REVIEW

**Comment by henryfan03** [comment]
awsuploads
postrequesttemplate

File: README.md [hunk 1]
```diff
@@ -47,3 +53,16 @@ Stop/Down docker containers
 docker-compose stop # pauses containers
 docker-compose down # brings down containers
 ```
+
+File S3 Object Requests Format
```
Comment (EricLi-Dev) [inline README.md:57]: This should probably be documented in the respective file itself as something like a docustring example, etc.

File: server/routes/file.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const express = require("express");
+const router = express.Router();
+const uploadImage = require("../utilities/uploadImage");
+
+//--File Endpoints (CRUD)--
+
+// update profile image
```
Comment (EricLi-Dev) [inline server/routes/file.js:7]: Better documentation is prob needed for each endpoint we have. If u can add that example S3 request here with descriptions for params and desc. for what the endpoint does

**Review by EricLi-Dev — APPROVED** [review]
Approving for now, but should also revert merge like #46 after pres. to fix up changes before actual merge to dev

**Review by EricLi-Dev — APPROVED** [review]

**Review by jubuyer — APPROVED** [review]
Looks good.

**Pull request merged**